src/app/common/money/format-money.ts

```diff
@@ -6,6 +6,10 @@ export function formatMoney({ amount, symbol, decimals }: Money) {
   return `${amount.shiftedBy(-decimals).toString()} ${symbol}`;
 }
 
```

Comment:
Maybe we can just make `symbol`  optional in `formatMoney` and avoid doubling up this logic?

Reply:
We try not to use optional parameters. I actually did this before here, but was asked to refactor it into a separate function.

Reply:
thanks for the info! 😄